src/main/java/com/glucoseguardian/webbackend/application/adapter/LoginAdapter.java

```diff
@@ -0,0 +1,78 @@
+package com.glucoseguardian.webbackend.application.adapter;
+
+import com.glucoseguardian.webbackend.application.service.LoginService;
+import com.glucoseguardian.webbackend.storage.dto.LoginInputDto;
+import com.glucoseguardian.webbackend.storage.dto.LoginOutputDto;
+import com.glucoseguardian.webbackend.storage.dto.MessaggioDto;
+import com.glucoseguardian.webbackend.storage.dto.TotpDto;
+import com.glucoseguardian.webbackend.storage.entity.Admin;
+import com.glucoseguardian.webbackend.storage.entity.Dottore;
+import com.glucoseguardian.webbackend.storage.entity.Paziente;
+import com.glucoseguardian.webbackend.storage.entity.Tutore;
+
+public class LoginAdapter implements LoginAdapterInterface {
+  private LoginService adaptee;
+
+  @Override
+  public LoginOutputDto login(LoginInputDto input) {
+    String email = input.getEmail();
+    String password = input.getPassword();
+    Integer otp = input.getOtp();
+
+    String idUtente;
+    int tipoUtente;
+
+    Admin adminRes = adaptee.loginAdmin(email, password, otp);
```

Comment:
Maybe we should have a UserDAO with a findColumnByEmail that returns if the searched email is connected to a Tutore, an Admin, a Paziente or a Dottore; or we should separate the different login, this is bad. (at least separate the admin and dottore ones who will be used in completely separate scopes).